ref/haskell/test/Spec.hs

```diff
@@ -22,6 +22,18 @@ validChecksums = map BSC.pack
     , "split1checkupstagehandshakeupstreamerranterredcaperred2y9e3w"
     ]
 
+invalidChecksums :: [BS.ByteString]
+invalidChecksums = map BSC.pack
+    [ " 1nwldj5"
+    , "\x7F" ++ "1axkwrx"
```

Comment:
In principle this should be `"\x7F\&1axkwrx"` or `"\DEL1axkwrx"`, not that it matters too much.